src/app/modules/shared_models/emoji_reactions_item.nim

```diff
@@ -0,0 +1,25 @@
+import NimQml
+
+type
```

Comment:
Minor: indention is not matching the code-style here, we using 2 spaces instead of 4